From 5d71fbcd3c2e425c9b42ec027318a569088ed8e1 Mon Sep 17 00:00:00 2001
From: aalbert <albert@physik.rwth-aachen.de>
Date: Fri, 20 Oct 2017 22:56:18 +0200
Subject: [PATCH 2/2] Reduce sampling number

---
 Template/LO/Source/dsample.f | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Template/LO/Source/dsample.f b/Template/LO/Source/dsample.f
index c805932..fa75768 100644
--- a/Template/LO/Source/dsample.f
+++ b/Template/LO/Source/dsample.f
@@ -1511,7 +1511,7 @@ c     Constants
 c
       include 'genps.inc'
       integer    max_events
-      parameter (max_events=500000) !Maximum # events before get non_zero
+      parameter (max_events=50000) !Maximum # events before get non_zero
 c
 c     Arguments
 c
-- 
2.7.4

